apps/nextjs-website/src/components/molecules/QuestionsAndAnswers/QuestionsAndAnswers.tsx

```diff
@@ -0,0 +1,81 @@
+'use client';
+import React, { useState } from 'react';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Button,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import { Compress, Expand, ExpandMore } from '@mui/icons-material';
+import { useTranslations } from 'next-intl';
+import { QuestionsAndAnswers } from '@/lib/types/webinar';
+import BlocksRendererClient from '../BlocksRendererClient/BlocksRendererClient';
+
+const MIN_QUESTIONS_TO_SHOW = 5;
+const MAX_QUESTIONS_TO_SHOW = 10;
+
+export type QuestionsAndAnswersProps = {
+  readonly questions: QuestionsAndAnswers[];
+};
+
+const QuestionsAndAnswersComponent = ({
+  questions,
+}: QuestionsAndAnswersProps) => {
+  const theme = useTheme();
+  const t = useTranslations('webinar.webinarsSection.questionsAndAnswers');
+  const [showMore, toggleShowMore] = useState(false);
+  const questionsToShow = showMore
+    ? MAX_QUESTIONS_TO_SHOW
+    : MIN_QUESTIONS_TO_SHOW;
+
+  const [expanded, setExpanded] = React.useState<number | false>(false);
+
+  const handleChange =
+    (panel: number) => (event: React.SyntheticEvent, newExpanded: boolean) => {
+      setExpanded(newExpanded ? panel : false);
+    };
+
+  return (
+    <Box pt={10} pb={10} sx={{ backgroundColor: theme.palette.grey[50] }}>
+      <Box sx={{ maxWidth: '1200px', margin: 'auto' }}>
+        <Typography variant='h4' sx={{ mb: 4, width: '100%' }}>
+          {t('title')}
+        </Typography>
+        {[...questions].splice(0, questionsToShow).map((question, index) => (
+          <Accordion
+            sx={{ marginBottom: 2, borderTop: 'none' }}
+            key={question.question}
+            disableGutters
+            expanded={expanded === index}
+            onChange={handleChange(index)}
+          >
+            <AccordionSummary expandIcon={<ExpandMore color='primary' />}>
+              <Typography>{question.question}</Typography>
+            </AccordionSummary>
+            <AccordionDetails>
+              <BlocksRendererClient content={question.answer} />
+            </AccordionDetails>
+          </Accordion>
```

Comment:
I see a lot of differences when comparing the component on Storybook with the design on Figma:
- the font size of the questions
- the font weight of the questions
- the vertical padding around questions
- the corners not rounded of the summary area
- the margin at the bottom of the accordion details
- the right distance of the arrow
- the font weight of the label "mostra più domande"
- the margins around the answers

![Screenshot from 2024-07-23 18-08-07](https://github.com/user-attachments/assets/95c89d29-176a-40dc-bcc4-adc3c06e98af)
![Screenshot from 2024-07-23 18-42-57](https://github.com/user-attachments/assets/d95cee1e-863d-476e-a66d-87f90e970b0c)
![Screenshot from 2024-07-23 18-24-59](https://github.com/user-attachments/assets/006c12fe-179e-4318-9a8a-e609b0421efe)
![Screenshot from 2024-07-23 18-22-38](https://github.com/user-attachments/assets/f6f0a777-9e10-489c-b09f-55f904fc9d86)
![Screenshot from 2024-07-23 18-20-36](https://github.com/user-attachments/assets/890247d0-f289-466b-b0a9-1bd80aa477da)
![Screenshot from 2024-07-23 18-18-26](https://github.com/user-attachments/assets/46410a1f-0c80-41ca-9201-2100943ce4de)